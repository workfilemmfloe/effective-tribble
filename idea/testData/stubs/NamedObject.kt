class A
trait T
object Test: A(), T {
}