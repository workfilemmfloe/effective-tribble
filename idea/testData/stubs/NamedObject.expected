PsiJetFileStubImpl[package=]
  PACKAGE_DIRECTIVE:
  CLASS:[fqName=A, isEnumEntry=false, isLocal=false, isTopLevel=true, isTrait=false, name=A, superNames=[]]
  CLASS:[fqName=T, isEnumEntry=false, isLocal=false, isTopLevel=true, isTrait=true, name=T, superNames=[]]
  OBJECT_DECLARATION:[fqName=Test, isCompanion=false, isLocal=false, isObjectLiteral=false, isTopLevel=true, name=Test, superNames=[A, T]]
    DELEGATION_SPECIFIER_LIST:
      DELEGATOR_SUPER_CALL:
        CONSTRUCTOR_CALLEE:
          TYPE_REFERENCE:
            USER_TYPE:[isAbsoluteInRootPackage=false]
              REFERENCE_EXPRESSION:[referencedName=A]
      DELEGATOR_SUPER_CLASS:
        TYPE_REFERENCE:
          USER_TYPE:[isAbsoluteInRootPackage=false]
            REFERENCE_EXPRESSION:[referencedName=T]
    CLASS_BODY:
